### PR TITLE
feat: engine dashboard

### PR DESCRIPTION
**Pull request by barak007**
This PR implements a new simple dashboard for the engine

### CODE REVIEW

File: packages/engine-cli/src/engine-build.ts [hunk 1]
```diff
@@ -176,17 +177,33 @@ export async function runEngine({
             console.log(`Runtime options: ${JSON.stringify(Object.fromEntries(runtimeOptions.entries()), null, 2)}`);
         }
 
-        runManagerContext = runNodeManager({
```
Comment (idoros) [inline packages/engine-cli/src/engine-build.ts:179]: runNodeManager doesn't seems to be used anymore

File: packages/engine-cli/src/dashboard/dashboard.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        tr.appendChild(configName);
+        tr.appendChild(run);
+
+        featureTable.appendChild(tr);
+    });
+}
+
+function populateDatalistOptions(options: string[], id: string) {
+    const dataList = byId(id);
+    dataList.id = id;
```
Comment (idoros) [inline packages/engine-cli/src/dashboard/dashboard.ts:98]: getting by id and then re-setting it?

File: packages/engine-cli/src/engine-build.ts (hunk 1 above)
Comment (barak007) [reply packages/engine-cli/src/engine-build.ts:179]: removed

File: packages/engine-cli/src/dashboard/dashboard.ts (hunk 2 above)
Comment (barak007) [reply packages/engine-cli/src/dashboard/dashboard.ts]: removed

**Review by idoros — APPROVED** [review]

**Pull request merged**